### PR TITLE
Primary filename

### PR DESCRIPTION
**Pull request by snehagunduraoUL**

### CODE REVIEW

File: config/metadata.yml [hunk 1]
```diff
@@ -467,15 +467,15 @@ fields:
     label: Rights License
     predicate: DC:license
   
-  masterFilename:
+  primaryFilename:
     definition: "The name of the file (usually attached to this work) that is considered the master digital file to represent the work."
```
Comment (rschwab) [inline config/metadata.yml:471]: can you swap 'master' for 'primary' here too?

File: spec/fixtures/files/sample_metadata_schema.yml [hunk 2]
```diff
@@ -431,10 +431,10 @@ fields:
     label: Rights License
     predicate: DC:license
   
-  masterFilename:
+  primaryFilename:
     definition: "The name of the file (usually attached to this work) that is considered the master digital file to represent the work."
```
Comment (rschwab) [inline spec/fixtures/files/sample_metadata_schema.yml:435]: here's another leftover 'master'

**Review by rschwab — COMMENTED** [review]
Would this require a re-index? Please check if this is an indexed field, and if so, if we even have data in it.

File: config/metadata.yml (hunk 1 above)
Comment (snehagunduraoUL) [reply config/metadata.yml:471]: Looks like its not an indexed field. Am I missing something? looked at schema.rb

Comment (rschwab) [reply config/metadata.yml:471]: Sounds good, I doubted it was but wanted to be sure. Once 'master' is out of the descriptions I think this is ready.

Comment (rschwab) [reply config/metadata.yml:471]: Sorry, definitions, not descriptions. Line 471 & 435

**Pull request merged**